src/bridge/DelayBuffer.sol

```diff
@@ -192,7 +200,9 @@ library DelayBuffer {
         uint64 maxBuffer,
         uint64 amountPerPeriod,
         uint64 period,
-        uint64 roundOff
+        uint64 roundOff 
+        // @review would rather blend (amountPerPeriod, period) together to a fixed unit
+        // instead of needing to keep track of 3 terms (amountPerPeriod, period, roundOff)
```

Comment:
there would still be rounding error but should be small enough to be negligible
we can always round down if we care about sequencer abusing the rounding

Reply:
you mean something like setting the replenish rate in the form of 1/period ? Or even a simplification which works for both blocks and seconds like if period = 12, then there is no round off.

Reply:
yes, to have sth like
```
BASIS = 10000
amountPerSecondInBasis = 1000 # equals to 0.1s per 1 second
```
then when we calculate how much to replenish, we always around off. The rounding should be small enough that is negligible, and always work against the sequencer (i.e. sequencer cannot gain more delay buffer by abusing the rounding)